CONTRIBUTING.md

```diff
@@ -32,12 +32,18 @@ If you feel like giving a hand, here are some ways:
 
 If you make a pull request, please follow the guidelines below:
 
-- [Setup Optuna](#setup-optuna)
-- [Checking the Format, Coding Style, and Type Hints](#checking-the-format-coding-style-and-type-hints)
-- [Documentation](#documentation)
-- [Unit Tests](#unit-tests)
-- [Continuous Integration and Local Verification](#continuous-integration-and-local-verification)
-- [Creating a Pull Request](#creating-a-pull-request)
+- [Contribution Guidelines](#contribution-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+    - [Setup Optuna](#setup-optuna)
+    - [Checking the Format, Coding Style, and Type Hints](#checking-the-format-coding-style-and-type-hints)
+    - [Documentation](#documentation)
+    - [Unit Tests](#unit-tests)
+    - [Continuous Integration and Local Verification](#continuous-integration-and-local-verification)
+    - [Creating a Pull Request](#creating-a-pull-request)
+  - [Learning Optuna's Implementation](#learning-optunas-implementation)
```

Comment:
I personally prefer the previous one since this ToC tree is under the "Pull Request Guidelines" section. 

How about reverting this change? At least, it seems to be out of scope for this PR.